backend/utils.py

```diff
@@ -1,17 +1,19 @@
-import asyncio
-import json
+from typing import Any, Dict, Union
 
+# import asyncio
+# async def notify_users(payload, users=None):
+#     if users is None:
+#         users = WEBSOCKET_INFO_DICT.keys()
+#     await asyncio.wait([user.send(payload) for user in users])
 
-async def notify_users(payload, users=None):
-    if users is None:
-        users = WEBSOCKET_INFO_DICT.keys()
-    await asyncio.wait([user.send(payload) for user in users])
 
-
-def cleanup_redis_dict(dict_from_redis):
+def cleanup_redis_dict(dict_from_redis: Dict[bytes, bytes]) -> Dict[str, str]:
```

Comment:
the Dict[bytes, bytes] type should probably be factored out into scrum_types, and then eventually used in the redis client typeshed